lib/src/vector_math/quad.dart

```diff
@@ -88,4 +88,21 @@ class Quad {
     _point2.add(offset);
     _point3.add(offset);
   }
+
+  /// Returns a printable string
+  @override
+  String toString() => '[0] $_point0\n[1] $_point1\n'
+      '[2] $_point2\n[3] $_point3\n';
+
+  /// Check if two vectors are the same.
```

Comment:
```suggestion
  /// Check if two quads are the same.
```

Reply:
gah sorry, its obvious where I copied and pasted this from :)